bc_obps/service/application_access_service.py

```diff
@@ -47,22 +47,22 @@ def is_user_eligible_to_request_admin_access(
 
         Args:
             user_guid (uuid): The user for whom eligibility is being checked.
-            operator_id (uuid): The if of the operator to which admin access is being requested.
+            operator_id (uuid): The id of the operator to which admin access is being requested.
 
         Returns:
             True or raises an exception.
         """
         approved_admins = UserOperatorDataAccessService.get_admin_users(operator_id, UserOperator.Statuses.APPROVED)
         if approved_admins.filter(user_guid=user_guid).exists():
-            raise UserError("You are already an admin for this Operator!")
+            raise UserError("You are already an admin for this Operator.")
```

Comment:
I didn't mean you had to change it! These messages probably came from the designers